src/harness/reference_models/move_list_calc/move_list_example.py

```diff
@@ -0,0 +1,109 @@
+# This software was developed by employees of the National Institute
+# of Standards and Technology (NIST), an agency of the Federal
+# Government. Pursuant to title 17 United States Code Section 105, works
+# of NIST employees are not subject to copyright protection in the United
+# States and are considered to be in the public domain. Permission to freely
+# use, copy, modify, and distribute this software and its documentation
+# without fee is hereby granted, provided that this notice and disclaimer
+# of warranty appears in all copies.
+#
+# THE SOFTWARE IS PROVIDED 'AS IS' WITHOUT ANY WARRANTY OF ANY KIND,
+# EITHER EXPRESSED, IMPLIED, OR STATUTORY, INCLUDING, BUT NOT LIMITED
+# TO, ANY WARRANTY THAT THE SOFTWARE WILL CONFORM TO SPECIFICATIONS, ANY
+# IMPLIED WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE,
+# AND FREEDOM FROM INFRINGEMENT, AND ANY WARRANTY THAT THE DOCUMENTATION
+# WILL CONFORM TO THE SOFTWARE, OR ANY WARRANTY THAT THE SOFTWARE WILL BE
+# ERROR FREE. IN NO EVENT SHALL NIST BE LIABLE FOR ANY DAMAGES, INCLUDING,
+# BUT NOT LIMITED TO, DIRECT, INDIRECT, SPECIAL OR CONSEQUENTIAL DAMAGES,
+# ARISING OUT OF, RESULTING FROM, OR IN ANY WAY CONNECTED WITH THIS
+# SOFTWARE, WHETHER OR NOT BASED UPON WARRANTY, CONTRACT, TORT, OR
+# OTHERWISE, WHETHER OR NOT INJURY WAS SUSTAINED BY PERSONS OR PROPERTY
+# OR OTHERWISE, AND WHETHER OR NOT LOSS WAS SUSTAINED FROM, OR AROSE OUT
+# OF THE RESULTS OF, OR USE OF, THE SOFTWARE OR SERVICES PROVIDED HEREUNDER.
+#
+# Distributions of NIST software should also include copyright and licensing
+# statements of any third-party software that are legally bundled with
+# the code in compliance with the conditions of those licenses.
+
+#=============================================================================
+# Test move_list calculation.
+# Expected result is 'Move list output: [True, True, True, False, True, False]'
+#=============================================================================
+
+
+import json
+import os
+from pykml import parser
+import move_list
+import time
+from collections import namedtuple
+
+if __name__ == '__main__':
+
+    # Inputs
+    low_freq = 3600000000 	# low frequency of protection constraint (Hz)
+    high_freq = 3610000000  # high frequency of protection constraint (Hz)
```

Comment:
See some other comment. I dont think this will work with the rest of the code, as it supports only 1MHz bandwdith otherwise it introduces an offset.

Reply:
I think we are clear on this.

Reply:
Ok forget this comment.